openapi/components/schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml

```
@@ -0,0 +1,19 @@
+allOf:
+  - $ref: ../Credential.yaml
+  - type: object
+    description: Avalara credential.
+    required:
+      - apiToken
+    properties:
+      apiToken:
+        type: string
+        description: Avalara API token.
+        format: password
+        example: 0ab15e3c1cc608b6f592fbddaddeb8f2
+      _links:
+        type: array
+        description: Related resource links.
```

nitpick: suggestion to help make the description more concise
```suggestion
        description: Related links.
```